### ISSUE TITLE
dropWhile

### ISSUE THREAD

**Issue by mspoulsen** [TylorS167/167#12]
```javascript
import { dropWhile } from '167'

const foo = dropWhile(n => n !== 99, [1,2,3,4,5])

console.log(foo) // returns [1,2,3,4,5]. Expected []
```

**Comment by TylorS** [comment]
Hey Morten, the fix has been released in v0.40

**Comment by TylorS** [comment]
Woops, sorry, I read `dropWhile` as `takeWhile`, I guess this is still a bug

**Comment by TylorS** [comment]
Okay now it's fixed in v0.41 👍
